addons/phantom_camera/scripts/phantom_camera/phantom_camera_properties.gd

```diff
@@ -28,6 +28,7 @@ var follow_has_path_target: bool
 var follow_path_node: Node
 var follow_path_path: NodePath
 var follow_mode: Constants.FollowMode = Constants.FollowMode.NONE
+var follow_pixel_perfect : bool
```

Comment:
Would prefer this being moved to `PhantomCamera2D.gd` instead, as it will only be used in 2D scenes.